apps/web/src/components/activity/ExecutionDetailsAccordion.tsx

```diff
@@ -0,0 +1,56 @@
+import { Accordion, useMantineColorScheme } from '@mantine/core';
+import { ExecutionDetailsStatusEnum } from '@novu/shared';
+import styled from 'styled-components';
+
+import { ExecutionDetailsStepContent } from './ExecutionDetailsStepContent';
+import { ExecutionDetailsStepHeader } from './ExecutionDetailsStepHeader';
+
+import { colors, shadows } from '../../design-system';
+
+const calculateStyle = (theme) => {
```

Comment:
You could use Mantine's [createStyles](https://v4.mantine.dev/theming/styles-api/#styling-with-classnames) that passes the theme for you.